docs/credits.md

```diff
@@ -1357,6 +1357,8 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | NuGet.Configuration                               | NuGet, Octopus Deploy                                                     | [https://github.com/OctopusDeploy/NuGet.Client](https://github.com/OctopusDeploy/NuGet.Client)                                                                                                                              | [Apache-2.0](https://raw.githubusercontent.com/OctopusDeploy/NuGet.Client/dev/LICENSE.txt)                          |
 | NuGet.Protocol.Core.Types                         | NuGet, Octopus Deploy                                                     | [https://github.com/OctopusDeploy/NuGet.Client](https://github.com/OctopusDeploy/NuGet.Client)                                                                                                                              | [Apache-2.0](https://raw.githubusercontent.com/OctopusDeploy/NuGet.Client/dev/LICENSE.txt)                          |
 | NuGet.Protocol.Core.v3                            | NuGet, Octopus Deploy                                                     | [https://github.com/OctopusDeploy/NuGet.Client](https://github.com/OctopusDeploy/NuGet.Client)                                                                                                                              | [Apache-2.0](https://raw.githubusercontent.com/OctopusDeploy/NuGet.Client/dev/LICENSE.txt)                          |
+| Nuke.Common                                       | Matthias Koch, Ulrich Buchgraber, Sebastian Karasek                       | [https://nuke.build/](https://nuke.build/)                                                                                                                                                                                  | [MIT](https://www.nuget.org/packages/Nuke.Common/5.2.1/license)                                                     |
+| Nuke.OctoVersion                                  | Nuke.OctoVersion                                                          | []()                                                                                                                                                                                                                        | [Undefined]()                                                                                                       |
```

Comment:
@OctopusDeploy/eng-prod looks like the Nuke.OctoVersion library is missing the hooks for License information?

Reply:
`Nuke.OctoVersion` has been removed - we're just waiting for a few PRs to go through to fix up some stuff to remove it (in favour of the stuff built into Nuke itself)